src/Form/DataField/LargeInt.hpp

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright_License {
+
+  XCSoar Glide Computer - http://www.xcsoar.org/
+  Copyright (C) 2000-2021 The XCSoar Project
+  A detailed list of copyright holders can be found in the file "AUTHORS".
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+}
+*/
+
+#ifndef XCSOAR_DATA_FIELD_LARGEINT_HPP
+#define XCSOAR_DATA_FIELD_LARGEINT_HPP
+
+#include "Number.hpp"
+#include "time/PeriodClock.hpp"
+
+class DataFieldLargeInt final : public NumberDataField
+{
+private:
+  int value;
+  int col_width;
+  PeriodClock last_step;
+  int speedup;
+
+  mutable TCHAR output_buffer[OUTBUFFERSIZE + 1];
```

Comment:
I don't know, I copied this component from the DataFieldInteger (Form/Integer.hpp) and didn't change the variable definition. Up to now I never used this, only const, constexpr,... and so on. If it doesn't make sense in this context, I will remove it.

Reply:
Otherwise I cannot use the function _stprintf in functions GetAsString and GetAsDisplayString, because there is no real const TCHAR buffer[xx]. 
Either I use this mutable - or I have this functions redesigned completely - without connection to DataFieldInteger.
So I will leave it as it is...